jobs/opensearch/templates/config/opensearch-security/roles.yml.erb

```diff
@@ -377,5 +377,6 @@ cf_user:
     fls:
     allowed_actions:
     - "read"
+    - "create"
```

Comment:
I don't think we need this

Reply:
was just seeing that, it should be ready now